test/spec/reference-types/select.txt

```diff
@@ -2,69 +2,69 @@
 ;;; STDIN_FILE: third_party/testsuite/proposals/reference-types/select.wast
 ;;; ARGS*: --enable-reference-types
 (;; STDOUT ;;;
-out/test/spec/reference-types/select.wast:283: assert_trap passed: unreachable executed
```

Comment:
we have a lot of random numbers polluting our tests.. would be great to find ways in which we can make it less brittle?

Reply:
agreed, but it's the tradeoff for having the ability to automatically rebase tests. If we remove the random numbers, we'd probably have to do something more like LLVM.